sdk/tools/asmpp/tokenizer.hpp

```diff
@@ -0,0 +1,248 @@
+/*
+ * PROJECT:     ReactOS host tools
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Tokenizer class implementation
+ * COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+#include <string>
+#include <vector>
+#include <fstream>
+#include <regex>
+#include <ctime>
+
+#ifdef _MSC_VER
+#define throw __debugbreak(); throw
+#endif
```

Comment:
Maybe surround that with an `#if _DEBUG` too?
(also, should this be `#if` or `#ifdef` ?)

Reply:
I'll use `#if 0`, since it's really only when you actively debug it.